.github/ISSUE_TEMPLATE/release.md

```diff
@@ -2,17 +2,17 @@
 name: Release Ticket
 about: A ticket tracking each release
 ---
+# Release MM/DD/YYYY
 
-## Release MM/DD/YYYY
 Roll out [updates/release to production](https://github.com/GSA/datagov-deploy/wiki/Releases) so that the latest bug fixes and feature code is available.
 
-### Acceptance Criteria
+## Acceptance Criteria
```

Comment:
Despite what the linter says, these have to be second-level since the rendered template appears in a page that already has an H1.
```suggestion
### Acceptance Criteria
```